cli_reference/cli_manager/cli-manager-release-notes.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="cli-manager-release-notes_{context}"]
+= {cli-manager} release notes
+include::_attributes/common-attributes.adoc[]
+:context: cli-manager-release-notes
+
+toc::[]
+
+The {cli-manager} allows you to install CLI plugins in both connected and disconnected environments.
+
+These release notes track the development of the {cli-manager} for {product-title}.
+
+//For an overview of the {run-once-operator}, see xref   :../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {cli-manager}].
+
+[id="cli-manager-release-notes-0-1-0_{context}"]
+== {cli-manager} 0.1.0 Technical Preview
+
+Issued: 19 November 2024
+
+The following advisory is available for the {cli-manager} 1.0.0:
```

Comment:
Here you say 1.0.0, but above it's 0.1.0. Which is it?

Reply:
It's 0.1.0. I'll fix it.